client/asset/btc/btc.go

```diff
@@ -58,7 +58,7 @@ const (
 	// redeem transaction.
 	defaultRedeemConfTarget = 2
 
-	minNetworkVersion  = 210000
```

Comment:
Should I leave this one? There's even a 27.1 https://github.com/bitcoin/bitcoin/releases/tag/v27.1